app/assets/javascripts/components/autocomplete.js

```diff
@@ -133,5 +135,66 @@ window.GOVUK.Modules = window.GOVUK.Modules || {};
     $select.parentNode.removeChild($select)
   }
 
+  Autocomplete.prototype.initAutoCompleteSearchTopics = function () {
+    var millerColumns = document.querySelector('miller-columns')
+    var topics = millerColumns.taxonomy.flattenedTopics
+
+    var topicSuggestions = []
+
+    topics.forEach((topic, index) => {
+      topicSuggestions.push({
+        topic: topic,
+        highlightedTopicName: topic.topicName.replace(/<\/?mark>/gm, ''), // strip existing <mark> tags
+        breadcrumbs: topic.topicNames
+      })
+    })
+
+    if (!topicSuggestions) {
+      return
+    }
+
+    new window.accessibleAutocomplete({ // eslint-disable-line no-new, new-cap
+      id: 'topics-autocomplete',
+      name: 'topics-autocomplete',
+      element: this.$module,
+      minLength: 3,
+      autoselect: false,
+      source: function (query, syncResults) {
+        var results = topicSuggestions
+
+        syncResults(query
+          ? results.filter(function (result) {
+            var indexOf = result.topic.topicName.toLowerCase().indexOf(query.toLowerCase())
```

Comment:
minor, but could be nice to set a variable of topicName so we can reduce this method chain?

By the way, this is all looking good now 👍